### PR TITLE
bump ics23@v0.12.0

### PR DESCRIPTION
**Pull request by SuperFluffy**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ sha2 = { version = "0.10", optional = true }
 blake3 = { version = "1.4.0", optional = true, features = ["traits-preview"] } 
 hex = "0.4"
 tracing = "0.1"
-ics23 = { version = "0.11.0", optional = true}
+ics23 = { version = "0.12.0", optional = true}
```
Comment (erwanor) [inline Cargo.toml:41]: `tonic` bump LGTM: https://github.com/cosmos/ics23/releases/tag/rust%2Fv0.12.0

**Review by erwanor — APPROVED** [review]
Could you bump to `version = "0.11.0"` and I'll cut a release for it

File: Cargo.toml (hunk 1 above)
Comment (SuperFluffy) [reply Cargo.toml:41]: I am not sure what that means. Did you mean that upstream also bumped?

**Comment by SuperFluffy** [comment]
> Could you bump to `version = "0.11.0"` and I'll cut a release for it

Will do in follow-up. Clean commits, etc

**Pull request merged**